pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2598,7 +2600,7 @@ func createCloudInitISO(config types.DomainConfig) (*types.DiskStatus, error) {
 
 	ds := new(types.DiskStatus)
 	ds.ActiveFileLocation = fileName
-	ds.Format = "raw"
+	ds.Format = zconfig.Format_RAW
```

Comment:
Do you need a ToLower here as well?

Reply:
No.. Here it is no longer a string.. But the Const ( int )

Reply:
OK